test/unit/response-logger/index.js

```diff
@@ -37,7 +37,34 @@ test('should return body content', t => {
     env: {}
   }
 
-  const { body } = buildLogResult({ req, res })
+  const x = buildLogResult({ req, res })
```

Comment:
O x só é utilizado pra destructuring né? Se for o caso dá pra acessar o { body }  aqui já :D

Reply:
ih, acabo sobrando isso hudsahudsa mb